unlock-app/src/vendor.d.ts

```diff
@@ -3,5 +3,4 @@
 // most convenient way will not buy us any type safety.
 declare module 'react-jazzicon'
 
-// TODO: Remove this when we have a TS build of unlock-js
```

Comment:
#5443 should have removed the need for this now that we've updated.

Reply:
We don't need this in the `vendor` file anymore, although it does seem like TS isn't picking up the definitions from the package.